### PR TITLE
install mac and win pared down, link to d4mac, d4win, getting started…

### PR DESCRIPTION
**Pull request by londoncalling**
**\- What I did**
Updates related to incorporating Docker for Mac and Docker for Windows into the docset, revised Mac and Win install topics, linked out to D4Mac and Win, improved Getting Started and Learn by example.

**\- How to verify it**
Local docs builds

Signed-off-by: Victoria Bialas victoria.bialas@docker.com

### CODE REVIEW

**Comment by londoncalling** [comment]
@SvenDowideit @sfsmithcha @icecrime updates related to docs Getting Started tutorial, Learn by example, and Mac and Windows Docker installs.

Please give me an LGTM if it does, and merge for me? (I don't have merge on this repo.)

File: docs/getstarted/index.md [hunk 1]
```diff
@@ -10,15 +10,30 @@ description = "Getting started with Docker"
...
 +++
 <![end-metadata]-->
 
 
 # Get Started with Docker
 
-This tutorial is a for non-technical users who are interested in learning more about Docker. By following these steps, you'll learn fundamental Docker features while working through some simple tasks. You'll learn how to:
+This tutorial is a for non-technical users who are interested in learning more about Docker. By following these steps, you'll learn fundamental Docker features while working through some simple tasks.
+
+Depending on how you got here, you may or may not have already downloaded Docker for your platform and installed it. The next
```
Comment (thaJeztah) [inline docs/getstarted/index.md]: unfinished sentence; "the next"?

File: docs/getstarted/last_page.md [hunk 2]
```diff
@@ -30,11 +30,11 @@ Depending on your interest, the Docker documentation contains a wealth of inform
...
   <tr>
     <td class="tg-031e">More about Docker for Mac, features, examples, FAQs, relationship to Docker Machine and Docker Toolbox, and how this fits in the Docker ecosystem</td>
-    <td class="tg-031e">[Getting Started with Docker for Mac](/docker-for-mac/index.md)</td>
+    <td class="tg-031e">[Getting Started with Docker for Mac](/mac/index.md)</td>
   </tr>
   <tr>
-    <td class="tg-031e">More about Docker for Windows, More about Docker for Windows, features, examples, FAQs, relationship to Docker Machine and Docker Toolbox, and how this fits in the Docker ecosystem</td>
-    <td class="tg-031e">[Getting Started with Docker for Windows](/docker-for-mac/index.md)</td>
+    <td class="tg-031e">More about Docker for Windows, features, examples, FAQs, relationship to Docker Machine and Docker Toolbox, and how this fits in the Docker ecosystem</td>
+    <td class="tg-031e">[Getting Started with Docker for Windows](/windows/index.md)</td>
```
Comment (SvenDowideit) [inline docs/getstarted/last_page.md]: no, this needs to be `/docker-for-mac` - the checkers don't know about the not-yet-released paths, and will fail.

File: docs/installation/mac.md [hunk 3]
```diff
@@ -11,418 +11,46 @@ weight="-90"
...
-        export DOCKER_MACHINE_NAME="default"
-        # Run this command to configure your shell:
-        # eval "$(docker-machine env default)"
-
-4. Connect your shell to the `default` machine.
-
-        $ eval "$(docker-machine env default)"
-
-5. Run the `hello-world` container to verify your setup.
+Docker for Mac is our newest offering for the Mac. It runs as a native Mac application and uses <a href="https://github.com/mist64/xhyve/" target="_blank">xhyve</a> to virutalize the Docker Engine environment and Linux kernel-specific features for the Docker daemon.
```
Comment (thaJeztah) [inline docs/installation/mac.md]: s/virutalize/virtualize/

File: docs/tutorials/menu.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!--[metadata]>
++++
+aliases = [
+]
+title = "Learn by example"
+description = "Explains how to work with containers"
+keywords = ["docker, introduction, documentation, about, technology, docker.io, user, guide, user's, manual, platform, framework, home,  intro"]
```
Comment (SvenDowideit) [inline docs/tutorials/menu.md]: you're missing `type = "menu"`

File: docs/getstarted/menu.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<!--[metadata]>
++++
+aliases = [
+]
+title = "Get Started with Docker"
+description = "Docker Mac"
+keywords = ["beginner, getting started, Docker"]
```
Comment (SvenDowideit) [inline docs/getstarted/menu.md]: `type = "menu"`

File: docs/installation/mac.md [hunk 6]
```diff
@@ -11,418 +11,46 @@ weight="-90"
...
 
+* If you are new to Docker, try out the [Getting Started](../getstarted/index.md) tutorial for a hands-on tour, including using Docker commands, running containers, building images, and working with Docker Hub.
 
-## Learning more
+* You can find more extensive examples in [Learn by example](../tutorials/index.md) and in the [Docker Engine User Guide](../userguide/index.md).
 
-Use `docker-machine help` to list the full command line reference for Docker Machine. For more
-information about using SSH or SCP to access a VM, see the [Docker Machine
-documentation](https://docs.docker.com/machine/).
+* If you are interested in using the Kitematic GUI, see the [Kitematic user guide](https://docs.docker.com/kitematic/userguide/).
```
Comment (thaJeztah) [inline docs/installation/mac.md]: should probably be `/kitematic/userguide.md`

File: docs/installation/windows.md [hunk 7]
```diff
@@ -11,369 +11,41 @@ weight="-80"
...
-* Docker Machine for running the `docker-machine` binary
-* Docker Engine for running the `docker` binary
-* Kitematic, the Docker GUI
-* a shell preconfigured for a Docker command-line environment
-* Oracle VM VirtualBox
+## Docker for Windows
 
-Because the Docker daemon uses Linux-specific kernel features, you can't run
-Docker natively in Windows. Instead, you must use `docker-machine` to create and attach to a Docker VM on your machine. This VM hosts Docker for you on your Windows system.
+Docker for Windows is our newest offering for PCs. It runs as a native Windows application and uses Hyper-V to virutalize the Docker Engine environment and Linux kernel-specific features for the Docker daemon.
```
Comment (thaJeztah) [inline docs/installation/windows.md]: s/virutalize/virtualize/

File: docs/installation/windows.md [hunk 8]
```diff
@@ -11,369 +11,41 @@ weight="-80"
...
+* If you are new to Docker, try out the [Getting Started](../getstarted/index.md) tutorial for a hands-on tour, including using Docker commands, running containers, building images, and working with Docker Hub.
 
-	  Run the <a
-	  href="http://www.microsoft.com/en-us/download/details.aspx?id=592"
-	  target="_blank"> Microsoft® Hardware-Assisted Virtualization Detection
-	  Tool</a> and follow the on-screen instructions.
+* You can find more extensive examples in [Learn by example](../tutorials/index.md) and in the [Docker Engine User Guide](../userguide/index.md).
 
-3. Verify your Windows OS is 64-bit (x64)
+* If you are interested in using the Kitematic GUI, see the [Kitematic user guide](https://docs.docker.com/kitematic/userguide/).
```
Comment (thaJeztah) [inline docs/installation/windows.md]: same here: `/kitematic/userguide.md`

File: docs/getstarted/last_page.md (hunk 2 above)
Comment (thaJeztah) [inline docs/getstarted/last_page.md]: ah, yes! just asked @londoncalling on slack, thanks for confirming

File: docs/tutorials/menu.md (hunk 4 above)
Comment (londoncalling) [inline docs/tutorials/menu.md]: @SvenDowideit Okay, but I don't see `type - "menu"` in the Piñata `menu.md`s you created ...

Comment (SvenDowideit) [inline docs/tutorials/menu.md]: yes.

**Comment by SvenDowideit** [comment]
you need to fix:
```
18:32:30 * link error: (in page engine/installation/mac.md) /kitematic/userguide/
18:32:30 * link error: (in page engine/installation/windows.md) /kitematic/userguide/
```

and ignore
```
18:32:30 * link error: (in page engine/getstarted/step_one.md) /docker-for-windows/index.md
18:32:30 * link error: (in page engine/installation/windows.md) /docker-for-windows/index.md
18:32:30 * link error: (in page engine/getstarted/last_page.md) /docker-for-windows/index.md
18:32:30 * link error: (in page engine/getstarted/step_one.md) /docker-for-mac/index.md
18:32:30 * link error: (in page engine/installation/mac.md) /docker-for-mac/index.md
18:32:30 * link error: (in page engine/getstarted/last_page.md) /docker-for-mac/index.md
```

**Comment by londoncalling** [comment]
@SvenDowideit  ok I thought I fixed `) /kitematic/userguide/`, will re-check

File: docs/getstarted/menu.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+[menu.main]
+identifier = "tutorial_getstart_menu"
+type = "menu"
```
Comment (SvenDowideit) [inline docs/getstarted/menu.md]: this need to move above the `[menu.main]`

File: docs/tutorials/menu.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+[menu.main]
+identifier = "engine_learn_menu"
+parent = "engine_use"
+type = "menu"
```
Comment (SvenDowideit) [inline docs/tutorials/menu.md]: `type = "menu"` needs to be above the `[menu.main]'

File: docs/tutorials/menu.md [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+aliases = [
+]
+title = "Learn by example"
+description = "Explains how to work with containers"
+keywords = ["docker, introduction, documentation, about, technology, docker.io, user, guide, user's, manual, platform, framework, home,  intro"]
+type = "menu"
+[menu.main]
+identifier = "engine_learn_menu"
+parent = "engine_use"
+type = "menu"
```
Comment (SvenDowideit) [inline docs/tutorials/menu.md]: remove this second one.

**Comment by SvenDowideit** [comment]
The checker failures are (sadly) expected, just the duplicate `type = "menu"` needs removing.

**Comment by SvenDowideit** [comment]
ok, LGTM

**Pull request merged**